docs/sphinx/_toc.yml.in

```diff
@@ -3,29 +3,35 @@
 root: index
 subtrees:
   - entries:
-      - file: what-is-migraphx
-      - file: migraphx-driver
-        subtrees:
-          - entries:
-            - file: reference/driver-options
-      - file: dev/contributing-to-migraphx
-        subtrees:
-          - entries:
-            - file: dev/data
-            - file: dev/operators
-            - file: dev/program
-            - file: dev/targets
-            - file: dev/quantization
-            - file: dev/pass
-            - file: dev/matchers
-            - file: dev/tools
-            - file: dev/env_vars
-            
-  - caption: API Reference
+    - file: what-is-migraphx
+    
+  - caption: API reference
     entries:
     - file: reference/cpp
     - file: reference/py
 
+  - caption: Command-line tool
+    entries:
+    - file: migraphx-driver
+      subtrees:
+        - entries:
+          - file: reference/driver-options
+
+  - caption:  Tutorial
```

Comment:
dev/contributing documentation should not go under `Tutorial` heading. This dev/contributing is not for users of migraphx so it should not go under a tutorial heading.